pkgs/development/python-modules/ansible/2.1.nix

```diff
@@ -15,15 +15,15 @@
 }:
 
 let
-  jinja = jinja2.override rec {
+  jinja = jinja2.overrideDerivation (old: rec {
```

Comment:
check out the new `overridePythonAttrs`. It overrides the call to `buildPythonPackage`.

Reply:
Is this recommend inside python-modules? Also after all ansible should not be there after all?

Reply:
Overriding within a derivation should indeed be prevented inside `python-packages.nix` to avoid multiple versions. But when you do override, I think `overridePythonAttrs` is typically most useful now.

I agree that `ansible` doesn't belong there, and if it would be moved the overriding wouldn't be an issue.

Reply:
I have move out ansible out of `pythonPackages`. In the documentation it is explained, how one can get a new python derivation with `overridePythonAttrs`, how does this work with `buildPythonApplication`

Reply:
Exactly the same.